### PR TITLE
[docs] updated ycsb commands

### LINKED ISSUE #14006: [Docs] YCSB requires number of rows in usertable to be equal to recordcount parameter for successful run
### Description

Running the YCSB benchmark [using our documentation](https://docs.yugabyte.com/preview/benchmark/ycsb-ysql/#6-run-individual-workloads-optional) I ran into an issue where there were many **[READ-FAILED]** and **[UPDATE-FAILED]** results during my testing. After further investigation this was due to the LOAD phase of the YCSB run:

```
./bin/ycsb load yugabyteSQL -s \
      -P db.properties           \
      -P workloads/workloada     \
      -p recordcount=1000000     \
      -p operationcount=10000000 \
      -p threadcount=32          \
      -p maxexecutiontime=180
```

Due to the `maxexecutiontime` parameter above the run times out after 180 seconds. In this particular case the LOAD timed out prior to inserting all 1000000 rows called out by recordcount above. Due to this when we run one of the workloads:

```
./bin/ycsb run yugabyteSQL -s  \
      -P db.properties           \
      -P workloads/workloada     \
      -p recordcount=1000000     \
      -p operationcount=10000000 \
      -p threadcount=256         \
      -p maxexecutiontime=180
```

We see these **[READ-FAILED]** and **[UPDATE-FAILED]** results due to **X** rows being inserted during the load phase and the workload looking for the `recordcount` value of **1000000** rows. 

- Users can fix this by setting the `recordcount` parameter to a value of **X**. Meaning if your table has 700k rows after the load phase you should set `recordcount` to 700000. 

We can help users avoid this by:

1. Removing the `maxexecutiontime` parameter from the examples in the documentation
2. Calling out a disclaimer that users need to check that number of table rows is equal to the `recordcount`. 

I prefer the first option since it requires no work for the user. 

FYI this is not seen when running the benchmark using the `run_ysql.sh` script as called out [in our documentation](https://docs.yugabyte.com/preview/benchmark/ycsb-ysql/#4-run-the-benchmark) because it calls the load without the `maxexecutiontime` parameter. However this does not work on YB Managed, for example, without modifications.

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #14006 
@mrajcevic01 , I have removed the `maxexecutiontime` as per your suggestion. 

@netlify /preview/benchmark/ycsb-ysql/#6-run-individual-workloads-optional

- Updated all the files with the note and general edits to maintain consistency in preview, stable, 2.12, and 2.8.

### CODE REVIEW

**Review by mrajcevic01 — COMMENTED** [review]
Looks good. Do you think we should still add a disclaimer about the fact that **recordcount** will need to match the number of rows in the table? Removing **maxexecutiontime** helps users avoid the issue but they can still run into it.

**Comment by aishwarya24** [comment]
@mrajcevic01 - Yes I have added it as well. Please let me know if this fine. Ill copy it to other files as well. Thanks!

**Review by mrajcevic01 — CHANGES_REQUESTED** [review]
I think we should put this note in the next section `Run the workload as follows:` since that is when the user would run into issues. During the load stage we are just inserting data. This note is specific to once the data has been loaded and before one of the actual workloads is run

**Review by mrajcevic01 — APPROVED** [review]

File: docs/content/preview/benchmark/ycsb-ysql.md [hunk 1]
```diff
@@ -94,108 +84,102 @@ db.user=yugabyte
...
 
-```sql
-yugabyte=# \c ycsb
-```
+    ```sql
+    yugabyte=# CREATE DATABASE ycsb;
+    ```
 
-Create the table.
+1. Connect to the database** as follows:
```
Comment (ddhodge) [inline docs/content/preview/benchmark/ycsb-ysql.md:127]: what are the asterisks for

File: docs/content/preview/benchmark/ycsb-ycql.md [hunk 2]
```diff
@@ -39,28 +39,22 @@ type: docs
 
 </ul>
 
-{{< note title="Note" >}}
+This document describes how to a YCQL-specific binding to test the YCQL API using the YCSB benchmark.
```
Comment (ddhodge) [inline docs/content/preview/benchmark/ycsb-ycql.md:42]: how to use

File: docs/content/preview/benchmark/ycsb-ycql.md [hunk 3]
```diff
@@ -69,132 +63,122 @@ $ tar -zxvf ycsb.tar.gz
...
+          -p recordcount=1000000     \
+          -p operationcount=10000000 \
+          -p threadcount=256
+    ```
 
-When run on a 3-node cluster with each a c5.4xlarge AWS instance (16 cores, 32 GB of RAM and 2 EBS volumes) all belonging to the same AZ with the client VM running in the same AZ we get the following results:
+## Expected results
 
-### 1 Million Rows
+When run on a 3-node cluster with each a `c5.4xlarge` AWS instance (16 cores, 32 GB of RAM and 2 EBS volumes) all belonging to the same availability zone with the client VM running in the same availability zone we get the following results for _1 million rows_:
```
Comment (ddhodge) [inline docs/content/preview/benchmark/ycsb-ycql.md:181]:
```suggestion
When run on a 3-node cluster with each a `c5.4xlarge` AWS instance (16 cores, 32 GB of RAM, and 2 EBS volumes) all belonging to the same availability zone with the client VM running in the same availability zone, you get the following results for _1 million rows_:
```

File: docs/content/preview/benchmark/ycsb-ysql.md [hunk 4]
```diff
@@ -94,108 +84,102 @@ db.user=yugabyte
...
+          -P workloads/workloadb     \
+          -p recordcount=1000000     \
+          -p operationcount=10000000 \
+          -p threadcount=256
+    ```
+
+## Expected results
 
-### 1 Million Rows
+When run on a 3-node cluster with each node on a c5.4xlarge AWS instance (16 cores, 32 GB of RAM, and 2 EBS volumes), all belonging to the same availability zone with the client VM running in the same availability zone we get the following results for _1 million rows_:
```
Comment (ddhodge) [inline docs/content/preview/benchmark/ycsb-ysql.md:182]:
```suggestion
When run on a 3-node cluster with each node on a c5.4xlarge AWS instance (16 cores, 32 GB of RAM, and 2 EBS volumes), all belonging to the same availability zone with the client VM running in the same availability zone, you get the following results for _1 million rows_:
```

File: docs/content/preview/benchmark/ycsb-jdbc.md [hunk 5]
```diff
@@ -107,91 +108,90 @@ $ ./run_jdbc.sh --ip <ip> --recordcount <number of rows>
...
+          -p recordcount=1000000     \
+          -p operationcount=10000000 \
+          -p threadcount=256
+    ```
 
-When run on a 3-node cluster of `c5.4xlarge` AWS instances (16 cores, 32GB of RAM and 2 EBS volumes) all belonging to the same AZ with the client VM running in the same AZ, expect the following results:
+## Expected results
 
-**One Million Rows**
+When run on a 3-node cluster of `c5.4xlarge` AWS instances (16 cores, 32GB of RAM and 2 EBS volumes) all belonging to the same availability zone with the client VM running in the same availability zone, expect the following results for _1 million rows_:
```
Comment (ddhodge) [inline docs/content/preview/benchmark/ycsb-jdbc.md:194]:
```suggestion
When run on a 3-node cluster of `c5.4xlarge` AWS instances (16 cores, 32GB of RAM, and 2 EBS volumes) all belonging to the same availability zone with the client VM running in the same availability zone, expect the following results for _1 million rows_:
```

File: docs/content/preview/benchmark/ycsb-jdbc.md [hunk 6]
```diff
@@ -202,6 +202,6 @@ When run on a 3-node cluster of `c5.4xlarge` AWS instances (16 cores, 32GB of RA
 | Workload E | 16,642 | 15ms scan | Not applicable |
 | Workload F | 29,500 | 2ms | 15ms read-modify-write |
 
-### Additional Examples
+## Additional examples
 
-For additional examples, refer to [Example Using a YCSB Workload with Automatic Tablet Splitting](/preview/architecture/docdb-sharding/tablet-splitting/#example-using-a-ycsb-workload-with-automatic-tablet-splitting).
+For additional examples, refer to [YCSB workload with automatic tablet splitting example](../../architecture/docdb-sharding/tablet-splitting/#ycsb-workload-with-automatic-tablet-splitting-example).
```
Comment (ddhodge) [inline docs/content/preview/benchmark/ycsb-jdbc.md:207]: deeplink only works in preview. only one example?

**Review by ddhodge — COMMENTED** [review]
LGTM

File: docs/content/preview/benchmark/ycsb-ysql.md (hunk 1 above)
Comment (aishwarya24) [reply docs/content/preview/benchmark/ycsb-ysql.md:127]: Connect to the database was in bold earlier , forgot to remove this. Thanks. Will fix it.

**Pull request merged**